client-dotnet/src/EvoMaster.Client.Util/JsonInspector.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace EvoMaster.Client.Util
+{
+    public class JsonInspector
+    {
+        public static bool ContainsKeyValue<T>(string json, string key, T value)
+        {
+            if (string.IsNullOrWhiteSpace(key))
+                throw new ArgumentNullException("Title");
+
+            var matches = GetMatches(json, key);
+
+            if (!matches.Any())
+            {
+                key = ToOppositeCase(key);
+
+                matches = GetMatches(json, key);
+            }
+
+            return matches.Any(x => x.Equals(value.ToString()));
+        }
+
+        private static ICollection<string> GetMatches(string json, string key)
+        {
+            var regex = new Regex($"(?<=\"{key}\"\\ *:\\ *\")(?:\\\\\"|[^\"])*");
```

Comment:
explain the regex in a comment

furthermore, are there no common libraries in .Net that do this kind of JSON handling? wondering if there is any edge case this regex would not handle

Reply:
This class is removed in the latest push